### ISSUE TITLE
Runner rejects Promise with non-error

### ISSUE THREAD

**Issue by muratsu** [express-validator/express-validator#572]
In runner.js there's a Promise.reject method call with no arguments:

https://github.com/express-validator/express-validator/blob/96a393f143e7c4ce26676b3edd5f4924bcb661d9/check/runner.js#L15

If we change the default Promise construct to Bluebird, it prints warnings to the console about the usage of promise rejection without an error.

http://bluebirdjs.com/docs/warning-explanations.html#warning-a-promise-was-rejected-with-a-non-error

eg:
```javascript
// file1.js
// Change default Promise to Bluebird
const Bluebird = require('bluebird');
global.Promise = Bluebird;

// file2.js
router
  .route('/stats/:userId')
  // GET /stats/:userId
  .get([check('userId').isInt()], fooController.barBaz);
```

Is it desirable to stick with Promises/A+ or should we fix and play better with Bluebird?

**Comment by gustavohenke** [comment]
I would accept a PR if we can make code complexity and Bluebird compliance play along well.
Do you want to take a stab at it? 😉

**Comment by muratsu** [comment]
If you're OK with it, I can replace `return Promise.reject();` with `return Promise.reject(new Error());`. It will have no impact/side effects on existing lib consumers and will also silent Bluebird warnings.

**Comment by gustavohenke** [comment]
Oops, didn't mean to close this issue with f1f57d5!

**Comment by gustavohenke** [comment]
👋 this has been fixed with [v6.0.0](https://github.com/express-validator/express-validator/releases/tag/v6.0.0)!
